src/pizzaiolo/pizzaiolo.service.ts

```diff
@@ -204,5 +207,32 @@ export class PizzaioloService {
       created_at,
       number,
     })
+
+    this.prisma.events.create({
```

Comment:
Mais a baixo nesse arquivo ele cria uma in memory message, acho que já da pra matar e criar com prisma né?

Reply:
show, vou tirar